code/game/objects/items/pneumaticCannon.dm

```diff
@@ -339,3 +339,8 @@
 /obj/item/storage/backpack/magspear_quiver/PopulateContents()
 	for(var/i in 1 to 20)
 		new /obj/item/throwing_star/magspear(src)
+
+
```

Comment:
no

Reply:
lol

Reply:
oops forgot to clean this up from when i tried making this thing a pneumatic cannon in the first place